### DISCUSSION TITLE
No Conflict for Alpine?

### DISCUSSION THREAD

**Discussion by yabdab** [alpinejs/alpine#1592] [4. General]
What happens if you have 2 ( or more ) copies of alpine loaded into a page? 

Does Alpine not care or is there a "No Conflict" directive similar to what jQuery does?

Also, what if you have v2 and v3 loaded at same time?

_I sometime build third-party embeds for sites and do not always have control of the environment._

**Comment by KevinBatdorf** [comment]
Have you experimented with this? Its also a possible concern for me in the future so if you identify any issues I'll help find a solution. I guess it would be pretty simple to test out.

**Reply by yabdab** [reply]
No, not yet, but I will.  I will report back here if I see any "conflicts".
